{"status": "broken", "name": "用例标题：正确的用户信息（填写所有字段）", "parameters": [{"name": "expected", "value": "{'status': 200, 'message': 'OK'}"}, {"name": "test_input", "value": "{'username': 'SUE_ZK04_A01', 'confirm': 'Qq123456', 'roleId': 138, 'cnName': u'\\u7528\\u62371', 'phoneNumber': '13200000001', 'password': 'UXExMjM0NTY=', 'email': '1063514310@qq.com', 'enName': 'englishname'}"}, {"name": "title", "value": "'正确的用户信息（填写所有字段）'"}], "labels": [{"name": "story", "value": "新增用户信息接口"}, {"name": "feature", "value": "用户管理"}, {"name": "tag", "value": "@pytest.mark.usefixtures('userId')"}, {"name": "suite", "value": "test_set"}, {"name": "subSuite", "value": "TestUser"}, {"name": "parentSuite", "value": "testcase"}, {"name": "host", "value": "DESKTOP-M0PGOTF"}, {"name": "thread", "value": "4156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "testcase.test_set"}], "stop": 1628559347492, "testCaseId": "927452938cd34b471a14569bf7d98541", "historyId": "378dd6c5b900fdd1f2ad9c1330d72a66", "start": 1628559347492, "fullName": "testcase.test_set.TestUser#test_user_add", "statusDetails": {"message": "TypeError: 'NoneType' object has no attribute '__getitem__'", "trace": "@pytest.fixture(scope='session')\n    def userId():\n>       userId = Login().yt_login('易通')['data']['userId']\nE       TypeError: 'NoneType' object has no attribute '__getitem__'\n\ntestcase\\conftest.py:23: TypeError"}, "uuid": "1bef0069-a80e-4774-9744-9e23465aad34"}